### PR TITLE
Add 0.61.0

### PR DESCRIPTION
**Pull request by rickhanlonii**
## Overview

Adds the changelog for 0.61.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,102 @@
 # Changelog
 
+## 0.61.0
+
+This is a major release that includes the new reloading experiance Fast Refresh. It also removes the React `.xcodeproj`, fixes `use_frameworks!` for CocoaPods support, adds a `useWindowDimensions` hook, and upgrades to React 16.9.
```
Comment (diegolmello) [inline CHANGELOG.md]: Typo: `experience`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,102 @@
...
+### Added
+
+* Add Fast Refresh by default ([17f8e58](https://github.com/facebook/react-native/commit/17f8e58) by [@gaearon](https://github.com/gaearon))
+* Add `useWindowDimensions` hook to replace most `Dimensions` usage ([5ec382d](https://github.com/facebook/react-native/commit/5ec382d) by [@sahrens](https://github.com/sahrens))
+
+#### Android specific
+
+* Add exception in .gitignore for `debug.keystore` to the android template. ([d550256](https://github.com/facebook/react-native/commit/d550256) by [@bondehagen](https://github.com/bondehagen))
+* Add jitpack repository to template ([1a92cf9](https://github.com/facebook/react-native/commit/1a92cf9) by [@iyegoroff](https://github.com/iyegoroff))
+* Add documentation to TextInput's Flow types ([d00f088](https://github.com/facebook/react-native/commit/d00f088) by [@empyrical](https://github.com/empyrical))
```
Comment (TheSavior) [inline CHANGELOG.md]: This probably shouldn't be in Android Specific. Maybe just put it in changed or fixed?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,102 @@
...
+* Deprecate method UIManagerModule.playTouchSound() ([e3ec8db](https://github.com/facebook/react-native/commit/e3ec8db) by [@mdvacca](https://github.com/mdvacca))
+* Deprecate UIManager.measureLayoutRelativeToParent ([e42009b](https://github.com/facebook/react-native/commit/e42009b) by [@mdvacca](https://github.com/mdvacca))
+
+#### Android specific
+
+* DrawerLayoutAndroid drawerPosition now expects a string, number is deprecated ([305b0a2](https://github.com/facebook/react-native/commit/305b0a2) by [@TheSavior](https://github.com/TheSavior))
+
+### Removed
+
+* Remove ensureComponentIsNative.js ([9a31fa5](https://github.com/facebook/react-native/commit/9a31fa5) by [@TheSavior](https://github.com/TheSavior))
```
Comment (TheSavior) [inline CHANGELOG.md]: This is internal only

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,102 @@
...
+* Remove style property bitmask ([f33a623](https://github.com/facebook/react-native/commit/f33a623) by [@davidaurelio](https://github.com/davidaurelio))
```
Comment (TheSavior) [inline CHANGELOG.md]: This seems like it is internal only

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,102 @@
...
+* Pop frames correctly in console.error handler ([3eaf245](https://github.com/facebook/react-native/commit/3eaf245) by [@motiz88](https://github.com/motiz88))
+
+#### Android specific
+
+* Add missing Hermes include ([1db96a3](https://github.com/facebook/react-native/commit/1db96a3) by [@janicduplessis](https://github.com/janicduplessis))
+* Fix UIManager.measure to consider scale and rotation transforms ([28d5018](https://github.com/facebook/react-native/commit/28d5018) by [@floriancargoet](https://github.com/floriancargoet))
+
+#### iOS specific
+
+* Fixes iOS packager connection not work ([4ab9da1](https://github.com/facebook/react-native/commit/4ab9da1) by [@zhongwuzw](https://github.com/zhongwuzw))
```
Comment (TheSavior) [inline CHANGELOG.md]: Maybe change the grammar on this one

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,102 @@
...
+* CLI is now ^3.0.0-alpha.1 ([5edd1c6](https://github.com/facebook/react-native/commit/5edd1c6) by [@thymikee](https://github.com/thymikee))
+* Flow is now v0.104.0 ([59db059](https://github.com/facebook/react-native/commit/59db059) by [@mroch](https://github.com/mroch))
+* React is now at 16.9 ([40e8a5f](https://github.com/facebook/react-native/commit/40e8a5f), [0ccedf3](https://github.com/facebook/react-native/commit/0ccedf3) by [@TheSavior](https://github.com/TheSavior))
+* Use Metro for auto-collapsing internal stack frames ([77125a1](https://github.com/facebook/react-native/commit/77125a1) by [@motiz88](https://github.com/motiz88))
+* Move React error message formatting into ExceptionsManager ([2dadb9e](https://github.com/facebook/react-native/commit/2dadb9e) by [@motiz88](https://github.com/motiz88))
+* Improve VirtualizedList error message ([bef87b6](https://github.com/facebook/react-native/commit/bef87b6) by [@vonovak](https://github.com/vonovak))
+
+#### Android specific
+
+* Bump Hermes to v0.1.1 ([c21e36d](https://github.com/facebook/react-native/commit/c21e36d) by [@sunnylqm](https://github.com/sunnylqm))
```
Comment (tatchi) [inline CHANGELOG.md]: According to https://github.com/react-native-community/releases/issues/140#issuecomment-533006341, hermes is at v0.2.1

**Comment by grabbou** [comment]
Shipping 0.61, thanks for all the hard work on the changelog!

**Pull request merged**